src/Configuration/LocalPlatformSettings.cs

```diff
@@ -41,7 +41,7 @@ public string LocalTestingStaticTestDataPath
             }
         }
 
-        public string LocalFrontendHostname { get; set; }
+        public string LocalFrontendHostname { get; set; } = "localhost";
```

Comment:
This change is unrelated, but makes the "Pick frontend version" page **not crash** when running from `dotnet run` because this setting is empty.